_posts/2021-06-08-keys-and-addresses.markdown

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "Wallet Basics: Keys and Addresses"
+categories:
+- Getting started
+last_modified_at: 2021-06-09
+---
+
+Keys and addresses are at the heart of all information flow in Cardano
+and other blockchains. Keys prove ownership, consent and authenticity
+while addresses derived from these keys are destinations to hold facts
+and values.
+
+A powerful feature arises from the fact that these cryptographic
+entities can be created without connecting to or interacting with the
+blockchain network. The de-coupling of these entities from the network
+allows any one, even those without an internet connection, to create
+unique addresses that can receive transaction outputs. Subsequently all
+the facts and values in these addresses can only be unlocked by the keys
+associated with it, safeguarding them from theft or confiscation.
+
+We will be focusing on the address keys on the right section of the
+image below.
+
+![](https://github.com/ilap/ShelleyStuffs/raw/master/images/ShelleyKeyAndAddresses.png)
+
+Image courtesy of [ilap](https://github.com/ilap)
+
+## Keys
+
+Two main types of keys are used within Cardano:
+
+- Payment Keys: For creating payment addresses and signing transactions
+  to spend ADA/assets in these payment addresses
+- Staking Keys: For creating payment/staking addresses, delegating
+  stake, claiming ADA rewards from staking addresses and registering
+  stake pools
+
+Both types are asymmetric
+[Ed25519](https://en.wikipedia.org/wiki/EdDSA#Ed25519) key pairs
+consisting of a private signing key and a public verification key.
+
+> Outside of the testnets it is **EXTREMELY** important to safeguard
```

Comment:
Can we add the custom class attributes `class="media notice notice-danger"` to this blockquote along with some internals like the `<i class="icon_ribbon_alt"></i>` like https://learn.lovelace.academy/getting-started/keys-and-addresses/#keys

Reply:
Is this going to apply to every blockquote? If it is, I can add it automatically to _layouts/post.html, otherwise you would have to mix HTML in your markdown manually because it doesn't support classes

Reply:
Mixing custom HTML can be fine for now